### PR TITLE
chore(deps) bump openresty from 1.17.8.2 to 1.19.3.1

### PR DESCRIPTION
**Pull request by bungle**
### Summary

See: https://openresty.org/en/changelog-1019003.html

### CODE REVIEW

**Comment by hutchic** [comment]
https://github.com/Kong/kong/pull/6638/files#diff-dabc5a8a5864490c868ed63726a07b9424be17da0b56b37e0b38ee7a4d423bf3L11 -- line 11 of the `.requirements` file

File: spec/01-unit/04-prefix_handler_spec.lua [hunk 1]
```diff
@@ -862,7 +862,7 @@ describe("NGINX conf compiler", function()
             local handle = io.popen("ls -l " .. conf[prefix .. "ssl_cert_default" .. suffix])
             local result = handle:read("*a")
             handle:close()
-            assert.matches("-rw-r--r--", result, nil, true)
+            assert.matches("%-rw%-r[-w]%-r%-%-", result, nil, false)
```
[the remark this answers is not in the document]
Comment (javierguerragiraldez) [reply spec/01-unit/04-prefix_handler_spec.lua:865]: this patter matches either `-rw-r--r--` or `-rw-rw-r--`, no `w` for "other", just optional for "group".  i found the exact result depends on the system's umask, but i don't think it's critical to tighten it to be always `-rw-r--r--`

File: kong/init.lua [hunk 2]
```diff
@@ -630,11 +630,13 @@ end
...
 
 function Kong.ssl_certificate()
+  local ctx = ngx.ctx
+
   kong_global.set_phase(kong, PHASES.certificate)
 
-  -- this doesn't really work across the phases currently (OpenResty 1.13.6.2),
-  -- but it returns a table (rewrite phase clears it)
-  local ctx = ngx.ctx
+  --TODO: doesn't work
```
Comment (fffonion) [inline kong/init.lua:637]: Is this a WIP or is something we will not ship in 2.4? Asking this as I see openresty changelog include some fixes for ngx.ctx in ssl_certificate phase.

Comment (bungle) [reply kong/init.lua:637]: this pr has not cleaned up yet, yesterday we finally got needed stuff merged and test suite at least a bit more stable

**Review by dndx — APPROVED** [review]

**Pull request merged**